che-components.yaml

```diff
@@ -0,0 +1,155 @@
+    endpoints:
+      - name: 'theia-dev-flow'
+        public: true
+        targetPort: 3010
+        attributes:
+          protocol: http
+    containers:
+      - name: theia-dev
+        image: 'quay.io/eclipse/che-theia-dev:next'
+        commands:
+          - name: uname
+            workingDir: '$(project)'
+            command: ['uname', '-a']
+        mountSources: true
+        ports:
+          - exposedPort: 3010
+        memoryLimit: '2Gi'
+
+  - id: containers/buildah/1.14.0
+    displayName: Buildah Base Dev Plugin
+    description: Container to build docker images with buildah
+    icon: https://raw.githubusercontent.com/containers/buildah/v1.14.0/logos/buildah-logomark_large.png
+    repository: https://github.com/containers/buildah
+    firstPublicationDate: '2020-04-15'
+    containers:
+      - name: buildah-dev
+        image: 'quay.io/eclipse/che-buildah-base:1.14.0-b0c91d3'
+        mountSources: true
+        memoryLimit: '1Gi'
+
+  - id: eclipse/che-async-pv-plugin/nightly
+    displayName: Che Async Storage Service
+    description: Che Plug-in with service to provide abilty for backup/restore project source from async storage
+    repository: https://github.com/che-incubator/workspace-data-sync
+    firstPublicationDate: '2020-07-10'
+    endpoints:
+      - name: 'che-rsync-events'
+        public: true
+        targetPort: 4445
+        attributes:
+          protocol: ws
+          type: rsync
+          discoverable: false
+          secure: false
+          cookiesAuthEnabled: true
+    containers:
+      - name: che-rsync
+        image: 'quay.io/eclipse/che-sidecar-workspace-data-sync:latest'
+        memoryLimit: '512M'
+        lifecycle:
+          preStop:
+            exec:
+            command: ['/bin/sh', '-c', '/scripts/backup.sh']
+        mountSources: true
+        ports:
+          - exposedPort: 4445
+        env:
+          - name: RSYNC_PORT
+            value: '2222'
+  - id: eclipse/che-async-pv-plugin/1.0.0
+    displayName: Che Async Storage Service
+    description: Che Plug-in with service to provide abilty for backup/restore project source from async storage
+    repository: https://github.com/che-incubator/workspace-data-sync
+    firstPublicationDate: '2020-07-10'
+    endpoints:
+      - name: 'che-rsync-events'
+        public: true
+        targetPort: 4445
+        attributes:
+          protocol: ws
+          type: rsync
+          discoverable: false
+          secure: false
+          cookiesAuthEnabled: true
+    containers:
+      - name: che-rsync
+        image: 'quay.io/eclipse/che-sidecar-workspace-data-sync:latest'
+        memoryLimit: '512M'
+        lifecycle:
+          preStop:
+            exec:
+            command: ['/bin/sh', '-c', '/scripts/backup.sh']
+        mountSources: true
+        ports:
+          - exposedPort: 4445
+        env:
+          - name: RSYNC_PORT
+            value: '2222'
+  - id: eclipse/che-machine-exec-plugin/nightly
```

Comment:
I think we should have similar format of separator between components (use empty line or not). As for me to have empty line between components is better

Reply:
@svor I've applied https://jsonformatter.org/yaml-formatter let me know if it is better now

Reply:
@benoitf yes, it's better now!